content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,169 @@
+
+[Admin]
+
+Docs: provide a link to the updated docs
+
+### Save time upgrading to Sourcegraph 4.0 with multi-version upgrades
+With 3.42, we introduced an all-new analytics including [XYZ]. Our new analytics make it easier for admins to understand user engagement, measure efficiency in terms of time saved, and help calculate the value of utilizing Sourcegraph and its features. We’ve significantly improved our in-product analytics to include the projected value and dev-time-saved for all Sourcegraph features, the ability to export data, individual user usage table, and much more. 
+
+![Multi-version upgrades](https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.0/Multi-version%20Upgrades.gif "Multi-version upgrades")
+
+[Admin]
+
+Docs: provide a link to the updated docs
+
+
+### Quickly see the status on your repository and sync permissions
+With Sourcegraph 4.0, we have introduced more than X quality of life improvements to help admins understand the current state of repository and permissions syncing. This helps to ensure users have the appropriate to repositories so they can understand, fix, and automate the codebase, while admins can XYZ. 
+
+Media: provide a screenshot, gif, or video to help support the copy[Admin]
+
+[Admin]
+
+Docs: provide a link to the updated docs
+
+### Gain knowledge about precise code navigation with improved analytics
+Get a better understanding of precise navigation coverage and access precise code graph data configured for all of your repositories with the improved code navigation dashboard.
+
+Media: provide a screenshot, gif, or video to help support the copy
+
+[Admin]
+
+Docs: provide a link to the updated docs
+
+### Uncover developer time saved using Browser and IDE extensions 
+Admins can now understand the adoption and amount of time saved from using IDE and Browser extensions across your organization. Additionally, to make it faster and easier to roll out our VS Code extension organization wide, we've created an example Batch Change that adds the extension to your repository’s recommended extensions. Once added, users will be recommended the Sourcegraph extension when opening the repository if they don't have it installed already. We’ve also updated our instructions on how to automatically install our Browser extensions for users in your Google Workspace organization. 
+
+Media: provide a screenshot, gif, or video to help support the copy
+
+[Admin]
+
+Docs: provide a link to the updated docs
+
+### New version of the reference panel
+TBD
+Headline: [Do X] with [Z]
+Description: 2 - 4 sentences on the feature, functionality and benefits
```

Comment:
We've introduced a new version of the reference panel for an improved code navigation experience. This new version stacks definitions and references in the same tab, has a brand new preview panel, and adds a filtering option for ease of navigation.